cdap-cli/src/main/java/co/cask/cdap/cli/command/metadata/AddMetadataPropertiesCommand.java

```diff
@@ -34,6 +34,10 @@
 
   private final MetadataClient client;
 
+  public static final String ENTITY_DESCRIPTION_STRING = String.format("'<%s>' is of the form " +
+    "'<entity-type>:<namespace-id>.<entity-name>', where '<entity-type>' is one of " +
```

Comment:
Now we have a problem. We were going to change all `<*-id>` parameters to `<*-name>`...but that would create a problem here. We would end up with `<entity-name>` twice:
```
Gets the metadata of an entity. '<entity-name>' is of the form '<entity-type>:<namespace-id>.<entity-name>', where '<entity-type>' is one of 'artifact', 'app', 'dataset', 'program', 'stream', or 'view'.
```
So, I'm thinking that what we should do is only change from `-id` to `-name` for simple types (non-composite).
The composite types remain as `-id`.

And in this case, we have an `<entity-id>` and a `<program-id>`, where `<program-id>` is composed of
`<app-name>.<program-type>.<program-name>`.

So this code would become:
```
  public static final String ENTITY_DESCRIPTION_STRING = String.format("'<%s>' is of the form " +
    "'<entity-type>:<namespace-id>.<program-id>', where '<entity-type>' is one of " +
    "'artifact', 'app', 'dataset', 'program', 'stream', or 'view', and where '<program-id>' is composed of " +
    "'<app-name>.<program-type>.<program-name>', where '<program-type>' is one of "+
    "flow, mapreduce, service, spark, worker, or workflow." , ENTITY);
```
though is that correct? Should we say `<program-id>` _can_ be either `<program-name>` or composed ... ?